tests/test_text_extraction.py

```diff
@@ -112,3 +112,47 @@ def test_issue_2336():
     page = reader.pages[0]
     actual_text = page.extract_text()
     assert "Beira Rio" in actual_text
+
+
+def test_layout_mode_font_class_to_dict():
+    from pypdf._text_extraction._layout_mode._font import Font
+
+    font = Font("foo", space_width=8, encoding="utf8", char_map={}, font_dictionary={})
+    assert Font.to_dict(font) == {
+        "char_map": {},
+        "encoding": "utf8",
+        "font_dictionary": {},
+        "space_width": 8,
+        "subtype": "foo",
+        "width_map": {},
+    }
+
+
+@pytest.mark.enable_socket()
+def test_layout_mode_epic_page_fonts():
+    url = "https://github.com/py-pdf/pypdf/files/13836944/Epic.Page.PDF"
+    name = "Epic Page.PDF"
+    reader = PdfReader(BytesIO(get_data_from_url(url, name=name)))
+    with open(RESOURCE_ROOT / "Epic.Page.layout.txt", encoding="utf-8") as fp:
```

Comment:
```suggestion
    expected = RESOURCE_ROOT.joinpath("Epic.Page.layout.txt").read_text(encoding="utf-8")
```

As we already have `pathlib.Path` objects here, just re-use its functionality. Further usages below can be simplified in a similar manner.

Reply:
That is pretty nice!